test/prototype_transforms_kernel_infos.py

```diff
@@ -451,3 +451,778 @@ def reference_inputs_convert_color_space_image_tensor():
+        combinations_grid(
+            interpolation=[F.InterpolationMode.NEAREST, F.InterpolationMode.BILINEAR],
+            expand=[True, False],
+            center=[None, (0, 0)],
+        ),
+    ):
+        if params["center"] is not None and params["expand"]:
+            # Otherwise this will emit a warning and ignore center anyway
+            continue
+
+        for fill in [None, 0.5, [0.5] * image_loader.num_channels]:
+            yield ArgsKwargs(
+                image_loader,
+                angle=_ROTATE_ANGLES[0],
+                fill=fill,
+                **params,
+            )
+
+
+def reference_inputs_rotate_image_tensor():
+    for image_loader, angle in itertools.product(make_image_loaders(extra_dims=[()]), _ROTATE_ANGLES):
+        yield ArgsKwargs(image_loader, angle=angle)
+
+
+def sample_inputs_rotate_bounding_box():
+    for bounding_box_loader in make_bounding_box_loaders():
+        yield ArgsKwargs(
+            bounding_box_loader,
+            format=bounding_box_loader.format,
+            image_size=bounding_box_loader.image_size,
+            angle=_ROTATE_ANGLES[0],
+        )
+
+
+def sample_inputs_rotate_mask():
+    for image_loader, params in itertools.product(
+        make_image_loaders(sizes=["random"], dtypes=[torch.uint8]),
+        combinations_grid(
+            expand=[True, False],
+            center=[None, (0, 0)],
+        ),
+    ):
+        if params["center"] is not None and params["expand"]:
+            # Otherwise this will emit a warning and ignore center anyway
+            continue
+
+        yield ArgsKwargs(
+            image_loader,
+            angle=_ROTATE_ANGLES[0],
+            **params,
+        )
+
+
+@pil_reference_wrapper
+def reference_rotate_mask(*args, **kwargs):
+    return F.rotate_image_pil(*args, interpolation=F.InterpolationMode.NEAREST, **kwargs)
+
+
+def reference_inputs_rotate_mask():
+    for mask_loader, angle in itertools.product(make_mask_loaders(extra_dims=[()], num_objects=[1]), _ROTATE_ANGLES):
+        yield ArgsKwargs(mask_loader, angle=angle)
+
+
+KERNEL_INFOS.extend(
+    [
+        KernelInfo(
+            F.rotate_image_tensor,
+            sample_inputs_fn=sample_inputs_rotate_image_tensor,
+            reference_fn=pil_reference_wrapper(F.rotate_image_pil),
+            reference_inputs_fn=reference_inputs_rotate_image_tensor,
+            closeness_kwargs=DEFAULT_IMAGE_CLOSENESS_KWARGS,
+        ),
+        KernelInfo(
+            F.rotate_bounding_box,
+            sample_inputs_fn=sample_inputs_rotate_bounding_box,
+        ),
+        KernelInfo(
+            F.rotate_mask,
+            sample_inputs_fn=sample_inputs_rotate_mask,
+            reference_fn=reference_rotate_mask,
+            reference_inputs_fn=reference_inputs_rotate_mask,
+            closeness_kwargs=DEFAULT_IMAGE_CLOSENESS_KWARGS,
+        ),
+    ]
+)
+
+_CROP_PARAMS = combinations_grid(top=[-8, 0, 9], left=[-8, 0, 9], height=[12, 20], width=[12, 20])
```

Comment:
@vfdev-5 The issue detailed in #6613 stems from these parameters. I just ported them 1-to-1 from our previous tests:

https://github.com/pytorch/vision/blob/841b9a19a5ae6fb92b517438061b9846153cc0c6/test/test_prototype_transforms_functional.py#L128

Should we change them?